app/Repositories/PhotoRepository.php

```diff
@@ -76,6 +76,72 @@ public function create(array $data, $signupId)
     }
 
     /**
+     * Update an existing photo post and signup.
+     *
+     * @param \Rogue\Models\Photo $signup
+     * @param array $data
+     *
+     * @return \Rogue\Models\Photo
+     */
+    public function update($signup, $data)
+    {
+        // Update the signup's quantity and why_participated data.
+        // We will always update these since we can't tell if this has been changed in a good way yet.
```

Comment:
Can you add a `@TODO` here to remove this logic when we are no longer supporting the phoenix-ashes campaign template